lib/utils/gestures.html

```diff
@@ -429,7 +429,7 @@
      * @memberof Polymer.Gestures
      * @param {!Node} node Node to add listener on
      * @param {string} evType Gesture type: `down`, `up`, `track`, or `tap`
-     * @param {!Function} handler Event listener function to call
+     * @param {!function(?Event):void} handler Event listener function to call
```

Comment:
Should this be `function(!Event):void`?

Reply:
yup i would think so. i set it as this because its what `_add` took